### PR TITLE
Change p2p_id to p2p_addr

### PR DESCRIPTION
**Pull request by evgeny-s**
[Ticket](https://cerenetwork.atlassian.net/browse/CBI-919)
* Renamed `p2p_id` to `p2p_addr`
* Changed `Vec<u64>` to `String`
* Renamved `ddn_id` to `p2p_id`

### CODE REVIEW

**Review by naure — APPROVED** [review]

File: cere02/lib.rs [hunk 1]
```diff
@@ -614,7 +614,7 @@ mod ddc {
     )]
     #[cfg_attr(feature = "std", derive(Debug, scale_info::TypeInfo))]
     pub struct DDCNode {
-        p2p_id: String,
+        p2p_addr: String,
         url: String,
     }
```
Comment (doasync) [inline cere02/lib.rs:619]: The events `DDCNodeAdded` and `DDCNodeRemoved` should be updated too

File: cere02/lib.rs [hunk 2]
```diff
@@ -640,7 +640,7 @@ mod ddc {
 
         /// Add DDC node to the list
         #[ink(message)]
-        pub fn add_ddc_node(&mut self, p2p_id: String, url: String) -> Result<()> {
+        pub fn add_ddc_node(&mut self, p2p_id: String, p2p_addr: String, url: String) -> Result<()> {
```
Comment (doasync) [inline cere02/lib.rs:643]: Shouldn't `p2p_addr` replace `p2p_id`?

Comment (evgeny-s) [reply cere02/lib.rs:643]: No, we have `p2p_id` in the first argument. We need both: `id` and `address`.
`Id` will be used as HashMap key.

File: cere02/lib.rs (hunk 1 above)
Comment (evgeny-s) [reply cere02/lib.rs:619]: Makes sense.

File: cere02/lib.rs [hunk 3]
```diff
@@ -767,7 +767,7 @@ mod ddc {
     )]
     #[cfg_attr(feature = "std", derive(Debug, scale_info::TypeInfo))]
     pub struct MetricKeyDDN {
-        ddn_id: Vec<u8>,
+        p2p_id: String,
```
Comment (doasync) [inline cere02/lib.rs:770]: It should be renamed here as well

File: cere02/lib.rs [hunk 4]
```diff
@@ -906,12 +906,12 @@ mod ddc {
         }
 
         #[ink(message)]
-        pub fn metrics_for_ddn(&self, ddn_id: Vec<u8>) -> Vec<MetricValue> {
+        pub fn metrics_for_ddn(&self, p2p_id: String) -> Vec<MetricValue> {
```
Comment (doasync) [inline cere02/lib.rs:909]: `p2p_addr`?

File: cere02/lib.rs (hunk 1 above)
Comment (evgeny-s) [reply cere02/lib.rs:619]: [Added](https://github.com/Cerebellum-Network/cere-enterprise-smart-contracts/pull/36/commits/d8308ef8df53946b0fb6987e4d07277b205c7d1c)

File: cere02/ddc/tests.rs [hunk 5]
```diff
@@ -1724,16 +1738,16 @@ fn report_metrics_updates_ddn_status_works() {
     let first_day = 1000;
 
     let today_ms = (first_day + 17) * MS_PER_DAY;
-    let ddn_id = b"12D3KooWPfi9EtgoZHFnHh1at85mdZJtj7L8n94g6LFk6e8EEk2b".to_vec();
+    let p2p_id = "12D3KooWPfi9EtgoZHFnHh1at85mdZJtj7L8n94g6LFk6e8EEk2b".to_string();
+    let p2p_addr = "/dns4/localhost/tcp/5000/p2p/12D3KooWPfi9EtgoZHFnHh1at85mdZJtj7L8n94g6LFk6e8EEk2b".to_string();
     let stored_bytes = 99;
     let wcu = 999;
```
Comment (doasync) [inline cere02/ddc/tests.rs:1744]: You can clarify that it's `wcu_used`

**Comment by doasync** [comment]
The code can be "prettified" using the following command:
`cargo +nightly fmt -- lib.rs ddc/tests.rs`

**Comment by evgeny-s** [comment]
> The code can be "prettified" using the following command:
I want to do it after we merge branches to `dev`. We have a lot of them, I want to prevent merging formatting.

File: cere02/ddc/tests.rs (hunk 5 above)
Comment (evgeny-s) [reply cere02/ddc/tests.rs]: [Renamed](https://github.com/Cerebellum-Network/cere-enterprise-smart-contracts/pull/36/commits/09b9e5e826e02d2aaf4453d83c6d9eb489f9a0ea)

**Review by doasync — APPROVED** [review]

**Pull request merged**